exporter/debugexporter/README.md

```diff
@@ -107,6 +108,38 @@ Attributes:
         {"kind": "exporter", "data_type": "traces", "name": "debug"}
 ```
 
+## Using the collector's internal logger
+
+When `use_internal_logger` is set to `true` (the default), the exporter uses the collector's [internal logger][internal_telemetry] for output.
+This comes with several consequences:
+
+1. The exporter's output is prefixed and suffixed by the internal logger's data.
+   - The prefix is the current timestamp and the `info` level marker, similar to `2023-11-10T22:49:03.510-0600    info`
+   - The suffix is the exporter's metadata in JSON format, similar to `{"kind": "exporter", "data_type": "traces", "name": "debug"}`
+1. The exporter's output is sent to the same destination as the collector's logs. By default, it is the [standard error][stderr] stream,
+   configurable via the [service.telemetry.logs.output_paths][configure_internal_logs] option.
+   It is not possible to send the Debug exporter's output to a different stream than the collector's internal logs.
+1. The output from the Debug exporter is sent to the internal logger with the log level `INFO`.
```

Comment:
Can the log level be made configurable?

Reply:
You mean when using the internal logger, the user might want to "log" the exporter's output with a higher log level, to prevent its output to be hidden when the user specified a higher log level on the collector? I suppose it might be a workaround for this issue.

I would not use this workaround, as I don't think the exporter should be using the internal logger.